test/yaml_md_test.ml

```diff
@@ -0,0 +1,94 @@
+open Olinkcheck
+
+let test_text_without_links () =
+  Alcotest.(check (list string))
+    "same lists" []
+    (YamlMd.extract_links
+       (YamlMd.from_string
+          "---\n\
+          \ field: a\n\
+          \ b: c\n\
```

Comment:
Is there an issue with character encoding here here?

Reply:
I don't think so? My editor is set to `utf-8`.